### DISCUSSION TITLE
Welcome to QHub OnPrem Discussions!

### DISCUSSION THREAD

**Discussion by dharhas** [nebari-dev/nebari-slurm#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by verhulstm** [comment]
Hi, Does this run on multiple VMs or physical nodes? or is it an AIO deployment?

**Reply by verhulstm** [reply]
say i have a vmware or openstack or pxe cluster, how do  i deploy qhub-hpc on that?

**Reply by costrouc** [reply]
Vagrant has support for multiple providers: https://www.vagrantup.com/docs/providers. I have several linux machines with kvm for testing so I've been leaning towards libvirt. Should work just fine on other virtualization providers. `qhub-hpc` is really just an ansible playbook. The vagrant bits are only for testing. The key requirement is 2+ nodes (1 master, 1 worker) running Ubuntu 20.04 where you can ssh as a non-root user with sudo.

**Comment by verhulstm** [comment]
using vagrant libvirt is interesting? why was this done?
